my-project/templates/product_category/_product_list.html.twig

```diff
@@ -0,0 +1,23 @@
+<table class="table table-hover table-striped">
```

Comment:
tables should be used to display content as its a very old approach, use div's

Reply:
@astyczen But bootstrap styles need <table> tag